src/__tests__/pages/onboarding/index.test.tsx

```diff
@@ -6,6 +6,9 @@ import renderWithProviders from "@/__tests__/__helpers__/renderWithProviders";
 jest.mock("posthog-js/react", () => ({
   useFeatureFlagEnabled: () => true,
 }));
+
+jest.mock("next/navigation", () => require("next-router-mock"));
```

Comment:
Isn't the mock router installed by default? I've been using it happily elsewhere.

Reply:
app router pages require a different router annoyingly, imported from `next/navigation` instead of `next/router`